services/static-webserver/client/source/class/osparc/wrapper/WebSocket.js

```diff
@@ -134,11 +134,13 @@ qx.Class.define("osparc.wrapper.WebSocket", {
       this.setNamespace(namespace);
     }
     this.__name = [];
+    this.__cache = {};
   },
 
   members: {
     // The name store an array of events
     __name: null,
+    __cache: null,
```

Comment:
My end goal is to create a kind of Crash Reporter where we collect and push some data to the backend (websocket messages, console errors, network errors...)

This will be enabled for all users, so for now, I will collect the socket messages for all roles.